lte/gateway/python/magma/pipelined/tests/snapshots/test_internal_pkt_ipfix_export.InternalPktIpfixExportTest.test_subscriber_policy.after_deletion.snapshot

```diff
@@ -14,11 +12,8 @@
  table=ue_mac(scratch_table_0), priority=15,tcp,tp_src=853 actions=set_field:0x1->reg6,resubmit(,ingress(main_table)),set_field:0->reg0,set_field:0->reg3
  table=ue_mac(scratch_table_0), priority=0 actions=resubmit(,ingress(main_table)),set_field:0->reg0,set_field:0->reg3
  table=ue_mac(scratch_table_1), priority=15 actions=CONTROLLER:65535,resubmit(,ingress(main_table))
- table=dpi(scratch_table_0), idle_timeout=42, priority=10,tcp,nw_src=1.2.3.0,nw_dst=45.10.0.1,tp_src=51115,tp_dst=80 actions=set_field:0xa->reg10,set_field:0->reg0,set_field:0->reg3
- table=dpi(scratch_table_0), idle_timeout=42, priority=10,tcp,nw_src=45.10.0.1,nw_dst=1.2.3.0,tp_src=80,tp_dst=51115 actions=set_field:0xa->reg10,set_field:0->reg0,set_field:0->reg3
- table=dpi(scratch_table_0), priority=0 actions=set_field:0xffffffff->reg10,set_field:0->reg0,set_field:0->reg3
- cookie=0xb, table=201, priority=0 actions=resubmit(,dpi(scratch_table_0)),resubmit(,202),set_field:0->reg0,set_field:0->reg3
+ cookie=0xb, table=201, priority=0 actions=resubmit(,202),set_field:0->reg0,set_field:0->reg3
```

Comment:
If a test fails because the snapshot is not equal to the state in the test, then a diff is printed - with this the snapshot can be updated. The review challenge is to agree/disagree that the removed flows only correspond to the removal of the dpi controller.